--- mod_authnz_external.c.orig	2012-08-24 12:49:58.000000000 +0400
+++ mod_authnz_external.c	2012-08-24 12:50:14.000000000 +0400
@@ -477,8 +477,8 @@
 	if (remote_host != NULL)
 	    child_env[i++]= apr_pstrcat(p, ENV_HOST"=", remote_host,NULL);
 
-	if (c->remote_ip)
-	    child_env[i++]= apr_pstrcat(p, ENV_IP"=", c->remote_ip, NULL);
+	if (c->client_ip)
+	    child_env[i++]= apr_pstrcat(p, ENV_IP"=", c->client_ip, NULL);
 
 	if (r->uri)
 	    child_env[i++]= apr_pstrcat(p, ENV_URI"=", r->uri, NULL);
